association_backend/association/domain/entities/subscription_entities.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
```

Comment:
about the file name: you don't need to repeat "entities", your file is already in a directory named entities

Reply:
same for the stripe file

Reply:
ok done